provider/status_changes.json

```diff
@@ -300,6 +300,28 @@
                 }
```

Comment:
Also just realized - this file is auto-generated. I believe the one you want to change is https://github.com/CityOfLosAngeles/mobility-data-specification/blob/dev/generate_schema/provider/status_changes.json